### PR TITLE
Add 2 NLI models and update the coref model

### PR DESCRIPTION
**Pull request by ZhaofengWu**
See #334

### CODE REVIEW

**Comment by dirkgr** [comment]
It won't run tests here of course. I didn't think of that. But tests would fail anyways because of all the broken models you didn't touch, right?

**Review by dirkgr — APPROVED** [review]

**Comment by ZhaofengWu** [comment]
I think so, but at least they will fail due to the fact that you are not
depending on a GitHub version of allennlp-models.

**Comment by dirkgr** [comment]
Can I just merge this, or do we need a review from @schmmd?

**Comment by ZhaofengWu** [comment]
I think that'll probably be better for him to review first unless you're submit a PR this week. Because otherwise if there're problems I won't be around.

**Review by schmmd — APPROVED** [review]

File: demo/src/components/demos/TextualEntailment.js [hunk 1]
```diff
@@ -166,7 +213,18 @@ const Attacks = ({attackData, attackModel, requestData}) => {
 
 
 const Output = ({ responseData, requestData, interpretData, interpretModel, attackData, attackModel}) => {
-  const { label_probs, h2p_attention, p2h_attention, premise_tokens, hypothesis_tokens } = responseData
+  const model = requestData ? requestData.model : undefined;
+
+  let label_probs, h2p_attention, p2h_attention, premise_tokens, hypothesis_tokens;
```
Comment (schmmd) [inline demo/src/components/demos/TextualEntailment.js:218]: This is unfortunate--but it's probably also what I would have done...

**Pull request merged**